i18n/en.pot

```diff
@@ -154,21 +157,3 @@ msgstr ""
 
 msgid "You are not authorized to use this app"
 msgstr ""
-
-msgid "Custom"
-msgstr ""
-
-msgid "Every hour"
-msgstr ""
-
-msgid "Every day midnight"
-msgstr ""
-
-msgid "Every day at three am"
-msgstr ""
-
-msgid "Every weekday at noon"
-msgstr ""
-
-msgid "Every week"
-msgstr ""
```

Comment:
Question - where did these strings come from and where did they go?  @ismay

Reply:
Hey, so these were part of an incomplete earlier fix. They're labels for the cron expression presets. The translations can also be found here:

https://github.com/dhis2/scheduler-app/blob/626e4ea32d12393941ff5a8f23ea1a83b29fc3c8/src/utils/apiTranslations.js#L10-L16

Those are the translations that this PR reverted back to, so the above (removed) translations could be safely removed.

Reply:
Ok, but does that mean these strings are only available in English and Urdu?  Is there a mechanism for hooking this up to Transifex?  Sorry if I'm missing some context here, not sure what mechanism of i18n this is using, want to make sure we have a full picture before the overhaul

Reply:
> Ok, but does that mean these strings are only available in English and Urdu?

That's right. At the moment there's only Urdu and English

>  Is there a mechanism for hooking this up to Transifex?

Don't know. This app hasn't been hooked up to transifex yet. Maybe we'll have to make some change before hooking it up to transifex.

> Sorry if I'm missing some context here, not sure what mechanism of i18n this is using, want to make sure we have a full picture before the overhaul

So what this PR does is just fix something that broke in the app's i18n setup. This isn't the refactored version, so I wouldn't pay too much attention to this version of the app. It'll be replaced soon anyway.

Reply:
Got it, thanks @ismay!

I'm going to make a note of it in dhis2/app-platform#369 so we don't forget to migrate it to the new system, since it might be easy to miss this non-standard i18n approach

Reply:
Thanks. Yeah it also caught us by surprise.